doc/content/source/problems/NekRSSeparateDomainProblem.md

```diff
@@ -1,33 +1,51 @@
 # NekRSSeparateDomainProblem
 
-This class allows for coupling of NekRS to a 1D T/H code such as SAM or THM. This coupling is performed using the "separate-domain" coupling strategy, where the coupling between codes is performed purely by
-updating boundary conditions between the domains. For most applications where there's a large flow loop, the 1D T/H code should model most of the loop, and NekRS should model only a portion of the loop that is of interest for CFD simulation.
+This class allows for coupling of NekRS to a 1D T/H code such as SAM or THM.
+This coupling is performed using the "separate-domain" coupling strategy,
+where the coupling between codes is performed purely by
+updating boundary conditions between the domains. For most applications where there's a large flow loop,
+ the 1D T/H code should model most of the loop, and NekRS should model only a portion of the loop that
+ is of interest for CFD simulation.
 
-The coupling's setup is controlled using the `coupling_type`, which provides information for how the NekRS is coupled to the 1D T/H code, via NekRS's `inlet`, `outlet`, or `inlet outlet`.
+The coupling's setup is controlled using the `coupling_type`, which provides information for how the
+ NekRS is coupled to the 1D T/H code, via NekRS's `inlet`, `outlet`, or `inlet outlet`.
 
 !alert warning
 This class currently only supports dimensional solutions coming from NekRS. Nondimensional support is in progress.
 
-### Velocity and temperature coupling
+### Velocity, temperature, and scalar coupling
 
-[inlet_outlet] shows the coupling of velocity and temperature for `coupling_type = inlet outlet`, where the 1D T/H code is coupled to NekRS's inlet and outlet boundaries.
+[inlet_outlet] shows the coupling of velocity (V), temperature (T), and scalar01 (S01)
+ for `coupling_type = inlet outlet`, where the 1D T/H code is coupled to NekRS's inlet and outlet boundaries.
```

Comment:
I think technically the syntax needs to have single quotes around the `MultiMooseEnum` if more than one is specified, i.e. `coupling_type = 'inlet outlet'`. I didn't catch this before, but let's update it now (there may be other places in the documentation where this happens too).